crates/typst/src/text/mod.rs

```diff
@@ -464,6 +464,21 @@ pub struct TextElem {
     #[ghost]
     pub hyphenate: Hyphenate,
 
+    /// Controls whether the word can be soft-wrapped between letters.
+    ///
+    /// ```example
+    /// #set page(width: 20pt)
+    /// 각 줄의 마지막에 한글이 올 때
+    /// 줄 나눔 기준을 “글자” 또는 “어절” 단위로 한다.
+    ///
+    /// #set text(word-break: "keep-all")
+    /// 각 줄의 마지막에 한글이 올 때
+    /// 줄 나눔 기준을 “글자” 또는 “어절” 단위로 한다.
+    /// ```
+    #[default(WordBreak::Normal)]
+    #[ghost]
+    pub word_break: WordBreak,
+
```

Comment:
I'd suggest starting with one or two examples in English, solely for consistency with the rest of the documentation. But keeping at least one CJK sample is also fine imo, however I don't know what the rules are here.